challenges/fizz_buzz_tree/README.md

```diff
@@ -0,0 +1,43 @@
+# FizzBuzzTree
+**Author** : Jay Adams
+**Version**: 0.1.0
+
+## Overview
+Travrese a BST using inorder, look for vals 5% 3% or both. replace val with new fizz, buzz or fizzbuzz 
+
+
+## Getting Started
+---------------
+- Change directory into your newly created project.
+    cd bst
+- Create a Python virtual environment.
+    python3 -m venv env
+- Upgrade packaging tools.
+env/bin/pip install 
+- Install the project in editable mode with its testing requirements.
+env/bin/pip install -e ".[testing]"
```

Comment:
Just so you know, you cannot `pip install` this project because it has not `setup.py` file.